### ISSUE TITLE
`StreamChannelException not defined`

### ISSUE THREAD

**Issue by cpm** [GetStream/stream-chat-ruby#40]
https://github.com/GetStream/stream-chat-ruby/blob/master/lib/stream-chat/channel.rb#L22

```ruby
    def url
      raise StreamChannelException 'channel does not have an id' if @id.nil?

      "channels/#{@channel_type}/#{@id}"
    end
```

When that line gets run, it raises:

> undefined method `StreamChannelException' for #<StreamChat::Channel:0x00007ffd46faff50>

instead of the intended exception.

I only came across it because I'm working on a Rails project that uses your gem and am using pretty imprecise `WebMock.stub_request` calls in my test suite. 

So, I might be hitting state that you would be incredibly unlikely to hit in the real world and maybe you don't have covered in your test suite. :)

**Comment by PWKad** [comment]
I have run in to this issue several times as well.

**Comment by ffenix113** [comment]
Thank you for reporting.

Resolved in v2.11.2
